tls/s2n_security_policies.c

```diff
@@ -855,6 +871,8 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version="test_ecdsa_priority", .security_policy=&security_policy_test_ecdsa_priority, .ecc_extension_required=0, .pq_kem_extension_required=0 },
     { .version="test_all_tls12", .security_policy=&security_policy_test_all_tls12, .ecc_extension_required=0, .pq_kem_extension_required=0 },
     { .version="test_all_tls13", .security_policy=&security_policy_test_all_tls13, .ecc_extension_required=0, .pq_kem_extension_required=0 },
+    { .version="test_all_equal_preference_tls13", .security_policy=&security_policy_test_all_equal_preference_tls13, .ecc_extension_required=0, .pq_kem_extension_required=0 },
```

Comment:
Just in general, we create test security policies only if we need them for integration tests(IIRC). Since your PR doesn't contain any integ tests, I wouldn't add these two to this PR. You can recreate them in a unit test if you need them for unit testing.

Reply:
Will move to unit test